core/src/main/java/org/commonjava/maven/ext/core/impl/RESTCollector.java

```diff
@@ -342,9 +343,9 @@ private static void recordPlugins( ManipulationSession session, Set<ArtifactRef>
 
         for ( ProjectVersionRef pvr : plugins.keySet() )
         {
-            deps.add( new SimpleArtifactRef(
+            deps.add( new SimpleScopedArtifactRef(
                             new SimpleProjectVersionRef( pvr.asProjectRef(), handlePotentialSnapshotVersion( vs, pvr.getVersionString() ) ),
-                            new SimpleTypeAndClassifier( "maven-plugin", null ) ) );
+                            new SimpleTypeAndClassifier( "maven-plugin", null ), ArtifactScopeEnum.compile.name() ) );
```

Comment:
I'm actually thinking this might be better left as null as it fits in with the defaults (empty == compile) and comparisons better.